website/templates/emails/email_contributors_of_registrations.txt.mako

```diff
@@ -0,0 +1,18 @@
+Dear ${user.fullname}, 
+
+You are receiving this message because you have an account on the Open Science Framework (OSF).  We are writing to let you know of a change to our model for registrations that may affect how you choose to use that feature.
+
+Registrations create a frozen, time-stamped snapshot of the project and its contents.  Registrations are used for archiving significant events in the research lifecycle, such as pre-registering an analysis plan prior to data collection, and they cannot be deleted.  Beginning on 29 May 2015, upon creating a registration, the user will choose to make it public immediately or set an embargo period keeping the project contents other than the title, description, creation date, embargo period, and contributors private for up to one year.  Users will be able to end the embargo early, but they will not be able to stop the registration from becoming public after the embargo.  Instead, to remove a public registration, users will have to retract it - similar to retraction of a published article.  The same meta-data that is available during the embargo would remain accessible, but all other project content would be inaccessible and replaced with a retraction notice.
+
+This change removes the possibility of having a permanently private registration.  The change also adds features to promote persistence and preservation.  OSF registrations will receive a persistent identifier (DOI), and registered projects will be archived by the OSF back-end to preserve the content.  Finally, when one project administrator registers the project, all contributors are notified of the registration and all project administrators will have 48 hours to cancel the registration before it is permanent.  We believe that these changes better aligns the registration function with the expectations of what registration should do.
```

Comment:
"These changes better aligns" -> "These changes better align"

I can make this change when I merge.